src/foam/nanos/crunch/ExpireUserCapabilityJunctionsCron.java

```diff
@@ -55,7 +55,9 @@ public void execute(X x) {
       } else {
         activeJunction.setIsInGracePeriod(true);
       }
-      if ( activeJunction.getStatus() == CapabilityJunctionStatus.EXPIRED ) activeJunction.clearData();
+      if ( activeJunction.getStatus() == CapabilityJunctionStatus.EXPIRED ) {
+        activeJunction.setData(null);
```

Comment:
so this would be migrated to 
activeJucntion.getPayload().clearData()
yes?
and any reason you wouldn't do that now?

Reply:
true, will update, thanks